src/current/v24.1/cutover-replication.md

```diff
@@ -154,16 +154,115 @@ At this point, the primary and standby clusters are entirely independent. You wi
 
 ## Cut back to the primary cluster
 
-After cutting over to the standby cluster, you may need to move back to the original primary cluster, or a completely different cluster. This process is manual and requires starting a new replication stream.
+After cutting over to the standby cluster, you may need to move back to the original primary cluster to serve your application.
 
-For example, if you had [set up physical cluster replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}) between a primary and standby cluster and then cut over to the standby, the workflow to cut back to the original primary cluster would be as follows:
+{% include {{ page.version.version }}/physical-replication/fast-cutback-syntax.md %}
 
-- Original primary cluster = Cluster A
-- Original standby cluster = Cluster B
+{{site.data.alerts.callout_info}}
+To move back to a different cluster, follow the physical cluster replication [setup]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}).
+{{site.data.alerts.end}}
+
+### Example
+
+The following is a step-by-step example of cutting back to the original primary cluster. This example assumes that a cutover from a primary to standby cluster has occurred, and the promoted standby has been serving traffic.
+
+- **Cluster A** = original primary cluster
```

Comment:
This may be easier to follow with more descriptive names, like `original_standby_cluster`, `original_primary_cluster`?